yarn-project/simulator/src/avm/journal/journal.ts

```diff
@@ -1,7 +1,8 @@
 import {
   AztecAddress,
-  type FunctionSelector,
+  FunctionSelector,
   type Gas,
+  PUBLIC_DISPATCH_SELECTOR,
```

Comment:
Unused now?

Just run yarn-format once you merge the PRs otherwise you'll get frustrated XD